src/internal/connector/sharepoint/list.go

```diff
@@ -86,6 +73,42 @@ func loadLists(
 	return results, nil
 }
 
+// fetchRelationships utility function to retrieve associated relationships:
+// - Columns, ContentTypes, ListItems
+func fetchRelationships(
```

Comment:
Nit: "relationships" is a bit ambiguous.
```suggestion
func fetchListContents(
```

Reply:
Resolved within [2f7fe31](https://github.com/alcionai/corso/pull/1788/commits/2f7fe310e3dd02babe7e8bd96cced47441bc90d3)